simulators/portal-interop/Dockerfile

```diff
@@ -18,12 +18,9 @@ FROM ubuntu:22.04
 
 RUN apt update && apt install wget -y
 
-# Use these for amd-based devices
-RUN wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4_amd64.deb
-RUN dpkg -i libssl1.1_1.1.0g-2ubuntu4_amd64.deb
-# Use these for arm-based devices
-#RUN wget http://ports.ubuntu.com/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_arm64.deb
-#RUN dpkg -i libssl1.1_1.1.1f-1ubuntu2_arm64.deb
+RUN export ARCH=$(uname -m | sed "s/aarch64/arm64/") \
```

Comment:
Did you test this on an x86_64 machine?

Because the resulting string does not work for me.  (uname = `x86_64`).

Reply:
I should probably stop commiting at 5 in a morning ;-)